app/frontend/src/components/ContributorForm/ContributorForm.tsx

```diff
@@ -139,7 +151,11 @@ export default function ContributorForm({ processForm }: ContributorFormProps) {
             {IDEAS_LABEL}
           </Typography>
           <TextField
-            inputRef={register}
+            inputRef={(el: HTMLInputElement) => {
+              if (!ideasInputRef.current && autofocus) el.focus();
+              if (el) ideasInputRef.current = el;
+              register(el);
+            }}
```

Comment:
I think it might be safer to put an if check on `el` to see if it exists first before calling `el.focus`, as [callback refs could get called with `null` first before receiving the `HTMLInputElement` instance](https://reactjs.org/docs/refs-and-the-dom.html#caveats-with-callback-refs) when using an inline function, so `el` should be really be `HTMLInputElement | null` (the MUI type here is a bit crap as it marks `el` with `any`...)